### PR TITLE
[v1.8.x] Enable CUDA 11.0 on nightly development builds

### PR DESCRIPTION
**Pull request by waytrue17**
## Description ##
This is a follow up PR to #19273. It enables CUDA 11.0 on nightly development builds.

### CODE REVIEW

**Comment by waytrue17** [comment]
@samskalicky @leezu can you pleas review? thanks

**Review by josephevans — APPROVED** [review]

**Review by samskalicky — APPROVED** [review]
Thanks @waytrue17 !

File: cd/README.md [hunk 1]
```diff
@@ -31,11 +31,11 @@ Currently, below variants are supported. All of these variants except native hav
 
 * *cpu*: CPU
 * *native*: CPU without MKL-DNN
-* *cu90*: CUDA 9.0
 * *cu92*: CUDA 9.2
 * *cu100*: CUDA 10
 * *cu101*: CUDA 10.1
 * *cu102*: CUDA 10.2
+* *cu110*: CUDA 11.20
```
Comment (szha) [inline cd/README.md:38]: ?

Comment (waytrue17) [reply cd/README.md]: Fixed the typo, thanks for pointing it out

**Comment by waytrue17** [comment]
@mxnet-bot run ci [centos-gpu,unix-cpu,windows-gpu]

**Comment by Light--** [comment]
how to install mxnet on cuda11 rtx 3090, is there any official documents?

**Pull request merged**